yarn-project/aztec.js/src/account/defaults/entrypoint_payload.ts

```diff
@@ -3,23 +3,30 @@ import { pedersenPlookupCompressWithHashIndex } from '@aztec/circuits.js/barrete
 import { padArrayEnd } from '@aztec/foundation/collection';
 import { FunctionCall, PackedArguments, emptyFunctionCall } from '@aztec/types';
 
-import partition from 'lodash.partition';
-
 // These must match the values defined in yarn-project/aztec-nr/aztec/src/entrypoint.nr
-export const ACCOUNT_MAX_PRIVATE_CALLS = 2;
-export const ACCOUNT_MAX_PUBLIC_CALLS = 2;
+export const ACCOUNT_MAX_CALLS = 4;
 
-/** Encoded payload for the account contract entrypoint */
-export type EntrypointPayload = {
+/** Encoded function call for account contract entrypoint */
+export type EntrypointFunctionCall = {
```

Comment:
This is nice 👍

Reply:
Yep, it was long due since Noir started supporting nested structs.